src/app/account/account-sidebar/account-users/account-users.component.html

```diff
@@ -11,27 +9,20 @@
     (opened)="openItem(user)"
   >
     <mat-expansion-panel-header>
-      <mat-panel-title>
+      <mat-panel-title class="entity-card-title" [matTooltip]="getUserName(user)">
         {{ user.firstname }} {{ user.lastname }}
```

Comment:
why do you use a function call in one place and not in the other one? use literal strings in both places or the function call in both places